### PR TITLE
[WIP] policy: Ask for permission if new rule exposes sub-resources

### PR DESCRIPTION
**Pull request by vadmeste**
Quick PR waiting for idea approval

### CODE REVIEW

File: cmd/policy-main.go [hunk 1]
```diff
@@ -253,12 +257,62 @@ func accessPermToString(perm accessPerms) string {
...
+	// Check if there is some overlapping existing rules
+	if !globalJSON {
+		rules, err := clnt.GetAccessRules(ctx)
+		if err != nil {
+			return err.Trace(targetURL)
+		}
+		path := strings.TrimPrefix(clnt.GetURL().Path, "/")
+		for k, v := range rules {
+			if strings.HasPrefix(k, path) && isPermHigher(accessPermToString(targetPERMS), v) {
+				if !askForConfirmation(fmt.Sprintf("You are exposing another sub-resource (`%s`, `%s`), do you confirm? ", k, v)) {
```
Comment (klauspost) [inline cmd/policy-main.go:309]: There should be an option to specify the answer from the commandline `--yes` `--no` or something like that for non-interactive mode.

Comment (vadmeste) [reply cmd/policy-main.go:309]: > There should be an option to specify the answer from the commandline `--yes` `--no` or something like that for non-interactive mode.

yes, it makes sense

**Comment by harshavardhana** [comment]
is this PR needed? @vadmeste

**Comment by vadmeste** [comment]
> is this PR needed? @vadmeste

No

**Pull request closed without merging**